sdk/peering/Azure.ResourceManager.Peering/tests/Scenario/PeeringServiceTests.cs

```diff
@@ -82,17 +80,26 @@ public async Task Delete()
             Assert.IsFalse(flag);
         }
 
-        [RecordedTest]
-        public async Task AddTag()
+        [TestCase(null)]
+        [TestCase(false)]
+        [TestCase(true)]
+        public async Task AddRemoveTag(bool? useTagResource)
         {
             string peeringServiceName = Recording.GenerateAssetName("peeringService");
```

Comment:
Please add this line at the beginning of this test:
```csharp
SetTagResourceUsage(Client, useTagResource);
```
This will change the behavior according the value of `useTagResource` to see if it can work as expected.

Reply:
need another pr